src/standard/fields.js

```diff
@@ -81,8 +81,8 @@ const fromTemplate = (template, options) => box => {
         : ['top', 'bottom', 'center'].includes(placement)
           ? extent[0] / 2
           : extent[0]
-
-      const text = (line, index) => makeText(x, index * dy, line)
+      const yOffset = options[style]['font-size']
+      const text = (line, index) => makeText(x, index * dy + yOffset, line)
```

Comment:
Since font-size is already referenced in line 73, I'm going to introduce it there and use it here as a fixed vertical offset (ignoring/removing dominant-baseline altogether).